onnxruntime/python/tools/transformers/onnx_model_bart.py

```diff
@@ -124,7 +125,12 @@ class BartOnnxModel(BertOnnxModel):
     def __init__(self, model, num_heads, hidden_size, model_impl="hf"):
         super().__init__(model, num_heads, hidden_size)
         self.attention_mask = AttentionMask(self)
-        self.attention_fusion = FusionBartAttention(self, self.hidden_size, self.num_heads, self.attention_mask)
+        if model_impl == "openai":
```

Comment:
I think we do not need model_impl. We can do like the following:
    fuse attention with FusionBartAttentionOpenai
    fuse attention with FusionBartAttention
    
So that it can handle graph patterns of both OpenAI and HF ones.